.github/workflows/main.yml

```diff
@@ -66,9 +66,7 @@ jobs:
       - uses: actions/checkout@v4
       - run: rustup update stable && rustup default stable
       - run: rustup component add clippy
-      # Only check cargo lib for now
-      # TODO: check every members
-      - run: cargo clippy -p cargo --lib --no-deps -- -D warnings
+      - run: cargo clippy --workspace --all-targets --no-deps -- -D warnings
```

Comment:
This is how fully redundant with setting `RUSTFLAGS: -D warnings` for tests.  Can we remove that?

The benefit will be is we'll get a full test report if we accidentally let a warning through.

Reply:
Did you mean removing `-- -D warnings`? I guess it is slightly different that we want to deny warnings for clippy lints.

Reply:
Yes, removing the `RUSTFLAGS`.  clippy should be a superset of the lints that the `RUSTFLAGS` is catching.